dbt_platform_helper/providers/ecs.py

```diff
@@ -13,7 +15,49 @@ def __init__(self, ecs_client, ssm_client, application_name: str, env: str):
         self.application_name = application_name
         self.env = env
 
-    def get_cluster_arn(self) -> str:
+    def start_ecs_task(
+        self,
+        cluster_name: str,
+        container_name: str,
+        task_def_arn: str,
+        vpc_config: Vpc,
+        env_vars: List[dict] = None,
+    ):
+        container_override = {"name": container_name}
+        if env_vars:
+            container_override["environment"] = env_vars
+
+        response = self.ecs_client.run_task(
+            taskDefinition=task_def_arn,
+            cluster=cluster_name,
+            capacityProviderStrategy=[
+                {"capacityProvider": "FARGATE", "weight": 1, "base": 0},
+            ],
+            enableExecuteCommand=True,
+            networkConfiguration={
+                "awsvpcConfiguration": {
+                    "subnets": vpc_config.public_subnets,
+                    "securityGroups": vpc_config.security_groups,
+                    "assignPublicIp": "ENABLED",
+                }
+            },
+            overrides={"containerOverrides": [container_override]},
+        )
+
+        return response.get("tasks", [{}])[0].get("taskArn")
+
+    def get_cluster_arn_by_name(self, cluster_name: str) -> str:
+        clusters = self.ecs_client.describe_clusters(
+            clusters=[
+                cluster_name,
+            ],
+        )["clusters"]
+        if len(clusters) == 1:
```

Comment:
It's possibly guaranteed by the api, but this would be safer as:
```suggestion
        if len(clusters) == 1 and "clusterArn" in clusters[0]:
```

Reply:
good catch, this should be more robust